content/react/concepts/hooks/hooks.md

```diff
@@ -11,10 +11,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-In React, hooks are functions that give function components class-like abilities. These abilities include:
-
-- Using `state`
-- Performing side effects
+In React, **hooks** are functions that give class-like abilities to function components, include [state](https://www.codecademy.com/resources/docs/react/state) and side-effects.
 
 There are a few rules when using hooks! Specifically:
```

Comment:
```suggestion
There are a few rules when using hooks, including the following:
```

Reply:
Can we also rephrase the two rules on lines # 18 and # 19? Maybe something like the following:

> \- Hooks must be called from React functions (i.e. components or custom hooks) and not from a regular JavaScript function.
> \- Hooks should not be called inside of a loop, condition, or nested function. Instead, hooks should be called at the top level of a React function so they render in the same order every time.